frontend/Styleguide/Button.tsx

```diff
@@ -20,23 +20,28 @@ export enum ButtonKind {
   secondary = "secondary",
   transparent = "transparent",
   classic = "classic",
+  icon = "icon",
 }
 
 export function buttonClasses(kind: ButtonKind, extraClassName: string = "") {
   const kindClasses = buttonKindClasses[kind] || "";
   return always(kindClasses)
     .always(extraClassName)
-    .maybe(styledButtonClasses, kind !== ButtonKind.classic)
+    .maybe(styledButtonClasses, styledButtonKinds.includes(kind))
```

Comment:
Nice addition here.  This way of writing the boolean seems to be more inclusive.